Index: src/main/java/problems_leet_code/removeDuplecates.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/problems_leet_code/removeDuplecates.java b/src/main/java/problems_leet_code/RemoveDuplecates.java
rename from src/main/java/problems_leet_code/removeDuplecates.java
rename to src/main/java/problems_leet_code/RemoveDuplecates.java
--- a/src/main/java/problems_leet_code/removeDuplecates.java	
+++ b/src/main/java/problems_leet_code/RemoveDuplecates.java	
@@ -1,16 +1,16 @@
 package problems_leet_code;
 
-public class removeDuplecates {
+public class RemoveDuplecates {
     public static void main(String[] args) {
         int arr[] = {0, 0, 0, 1, 2, 2, 3, 3, 4, 5, 6, 6, 45, 45, 50, 66, 66, 77};
-        int temp = 0;
+        int j = 0;
         for (int i = 0; i < arr.length - 1; i++) {
             if (arr[i] != arr[i + 1]) {
-                arr[temp] = arr[i];
-                temp++;
+                arr[j] = arr[i];
+                j++;
             }
         }
-        arr[temp++] = arr[arr.length - 1];
+        arr[j++] = arr[arr.length - 1];
 
         for (int i = 0; i < arr.length; i++) {
             System.out.print(arr[i] + " ");
